### DISCUSSION TITLE
Great project....bell

### DISCUSSION THREAD

**Discussion by pfuller88** [GerryDazoo/Slinger#206] [General]
I have two slingboxs in two locations.  M1 and solo.   Both are working locally and remotely with your software.   My solo is in Canada hooked up to Bell Fibe cable box.  I found the TIVO remote worked well, for most buttons.  I customized a little and added a few favorite channel buttons (labeled) and a 3 minite skip button.  A work in progress, attached the file.
[BellFibeRemote.txt](https://github.com/GerryDazoo/Slinger/files/9903734/BellFibeRemote.txt)

**Comment by Chaskel** [comment]
Hi @pfuller88 - I was just wondering if this file is still working for you? I noticed there was a "minor bug fix" version in https://github.com/GerryDazoo/Slinger/tree/main/CustomRemotes, but if I use either that file or the one you attached, I see the following when launching Slinger:

Reading Custom Remote definition from BellFibeRemote.txt
Error reading remote definition file, using defaults

**Reply by pfuller88** [reply]
yes it works fine. reminder to make sure you have the sub folder as well.

**Reply by pfuller88** [reply]
also, i created a mapped version for Bell, with the remote image file.  if i remember I will post tomorrow

**Reply by Chaskel** [reply]
Interesting. Presuming I have this correct, I have it placed in the CustomRemotes directory that is located off of the directory I have the slingbox_server.py file in, and have my .ini file configured with "Remote=BellFibeRemote.txt" (and I'm using the latest version of slingbox_server.py). If I comment out the custom remote line, I don't receive the error.

Also, thanks for the info on the mapped version. I look forward to hopefully seeing what you've got there.

**Reply by pfuller88** [reply]
your .ini file line should end in .html, not .txt.   e.g….  “Remote=BellFibeRemote.html”

**Reply by pfuller88** [reply]
i’ll upload the latest versions of both shortly.  The latest version of the above has a file name Remote=BellFibeRemotePF.html

**Reply by Chaskel** [reply]
Just to confirm, even though the file is called BellFibeRemote.txt, I should reference it as BellFibeRemote.html in the .ini file? (unless the .html one you are referring to is the one you mentioned providing today?)

**Reply by Chaskel** [reply]
> i’ll upload the latest versions of both shortly. The latest version of the above has a file name Remote=BellFibeRemotePF.html

Ok, thanks!

**Comment by pfuller88** [comment]
Ok, sorry, the .txt file in Gerry's remotes section is quite old.  Attached in the below zip are the 2 remotes I use for my Toronto based BellFibe (channels are mapped to Toronto's bell service).   The v2 has a readme file in the sub directory.  For both, download the files and subdirectories to your slinger folder and point the ini remote line to either of the .html files.

[BellFibeRemotes.zip](https://github.com/GerryDazoo/Slinger/files/10309985/BellFibeRemotes.zip)

**Reply by Chaskel** [reply]
Many thanks for posting. I will grab a copy and give it a go, and if I run into any issues I will let you know.

Take care, and happy new year.
